ProjectWork/Pages/Authentication/AccountEdit.razor

```diff
@@ -13,91 +16,85 @@
         <h3 class="w3-margin-left">User Profile</h3>
 
         <hr />
+        <EditForm Model="_editUser" OnValidSubmit="async ()=>await AttempEdit(_editUser)">
 
-        <ul>
-            <li>
-                <p class="w3-bold">Username:</p>
-            </li>
-            <li>
-                <input class="w3-round w3-margin-right w-50 w3-margin-bottom" type="text" id="username">
-            </li>
-            <li>
-                <p class="w3-bold">E-mail:</p>
-            </li>
-            <li>
-                <input class="w3-round w3-margin-right w-50 w3-margin-bottom" type="text" id="mail">
-            </li>
-            <li>
-                <p class="w3-bold">Bio:</p>
-            </li>
-            <li>
-                <textarea id="w3review" name="w3review" rows="4" cols="50">
-                </textarea>
-            </li>
-
-            <li>
-                <span class="w3-bold w3-large">Type of access: </span>
-                <span class="w3-text-green">Standard</span>
-            </li>
-            <li>
-                <span>
-                    <button onclick="@editPassword" class="w3-large btn w3-button w3-red w3-round w3-margin-bottom w3-margin-top w3-margin-right">
-                        Edit your Password
-                    </button>
-                </span>
-            </li>
-
-            @if (showEditPassword)
-            {
+            <ul>
                 <li>
-                    <p class="w3-bold">Old password:</p>
+                    <p class="w3-bold">Username:</p>
                 </li>
                 <li>
-                    <input class="w3-round w3-margin-right w-50 w3-margin-bottom" type="password" id="oldpw">
+                    <input @bind-value="_editUser.UserName"  
```

Comment:
Something nice to consider just for QoL would be nice to show the previous value:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/text#placeholder